content/en/tracing/connect_logs_and_traces/ruby.md

```diff
@@ -17,17 +17,20 @@ further_reading:
       text: 'Correlate request logs with traces automatically'
 ---
 
-## Automatic Trace ID injection
+## Automatic log injection
 
 Use one of the following options to inject Ruby trace information into your logs:
 
 - [Automatic Trace ID Injection for Rails Applications using Lograge is recommended](?tab=lograge).
 - [Automatic Trace ID Injection for default Rails Applications](?tab=default).
 
+If you haven't done so already, we recommend configuring the Java tracer with `DD_ENV`, `DD_SERVICE`, and `DD_VERSION`. This will provide the smoothest
```

Comment:
Although this is definitely our intention and future plan, this is currently misleading. We do in fact support `env`, `service` and `version` in Ruby, `service` is not fully implemented, and cannot be for some time because it involves breaking changes to many of our integrations and unresolved questions of what to do with integrations that have "external" services.

I wouldn't want users to read this today and assume that they can set `service` to see everything get the expected `service` value. They won't and they will likely open bug reports... I don't think we're ready to advertise this for Ruby.